pkg/config/config_template.yaml

```
@@ -1468,6 +1468,11 @@ api_key:
   ##        The set of keys for which their values will be passed through SQL obfuscation
   #         obfuscate_sql_values:
   #             - val1
+  #
+  #     command_execution:
+  ##        @param DD_APM_OBFUSCATION_COMMAND_EXECUTION_ENABLED - boolean - optional
+  ##        Enables obfuscation rules for spans of type "command_execution". Enabled by default.
+  #         enabled: false
```

nitpick: this should reflect the default
```suggestion
  #         enabled: true
```